document-generation/utils/utils.ts

```diff
@@ -84,35 +84,38 @@ interface PDFTemplateFiles {
 }
 
 // documents and the related templates
-const documentTemplatePaths: TemplatePaths = {
-  [DocumentType.DESCRIPTION_OF_WORK_PDF]: {
-    html: "/opt/dow-template.html",
-    css: "/opt/dow-style.css",
-  },
-  [DocumentType.DESCRIPTION_OF_WORK_DOCX]: {
-    docx: "/opt/dow-template.docx",
-  },
-  [DocumentType.INDEPENDENT_GOVERNMENT_COST_ESTIMATE]: {
-    excel: "/opt/igce-template.xlsx",
-  },
-  [DocumentType.INCREMENTAL_FUNDING_PLAN]: {
-    docx: "/opt/ifp-template.docx",
-  },
-  [DocumentType.EVALUATION_PLAN]: {
-    docx: "/opt/eval-plan-template.docx",
-  },
-  [DocumentType.REQUIREMENTS_CHECKLIST]: {
-    docx: "/opt/requirements-checklist-template.docx",
-  },
+export const getDocumentTemplatePath = (folder?: string): TemplatePaths => {
```

Comment:
Nice

Reply:
I have some code that I use for local testing that I don't want to commit as actual code but might just throw in the README later, but it uses this method and passes in its own folder.